templates/default/st2.conf.erb

```diff
@@ -65,12 +84,6 @@ remote_dir = /tmp
 [action_sensor]
 triggers_base_url = <%= @api_url %>/v1/triggertypes/
 
-[resultstracker]
-logging = <%= @conf_root %>/st2actions/console.conf
-
-[notifier]
-logging = <%= @conf_root %>/st2actions/console.conf
-
 [database]
```

Comment:
@armab Dunno if this is still relevant with `2.0.0` or we can get rid of this `database` category.